ddtrace/internal/_unpatched.py

```diff
@@ -10,9 +10,13 @@
 # to get a reference to the right threading module.
 import threading as _threading  # noqa
 import gc as _gc  # noqa
-
 import sys
 
+_Lock = _threading.Lock
+_RLock = _threading.RLock
+_Event = _threading.Event
```

Comment:
nit: maybe we can be super-explicit?

```suggestion
_Unpatched_Lock = _threading.Lock
_Unpatched_RLock = _threading.RLock
_Unpatched_Event = _threading.Event
```